framework/Util/TDbPluginModule.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * TDbPluginModule class file
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @link https://github.com/pradosoft/prado
+ * @license https://github.com/pradosoft/prado/blob/master/LICENSE
+ * @package Prado
+ */
+	
+namespace Prado\Util;
+
+use Prado\Util\XTPluginModule;
```

Comment:
This class doesn't exists, is it a leftover?

Reply:
This is a leftover.  I'll remove it.